frontend/src/App.tsx

```diff
@@ -4,6 +4,7 @@ import { createTheme, responsiveFontSizes, ThemeProvider } from "@mui/material";
 import theme from "config/theme.config.json";
 import { AppRoutes } from "routes";
 import { AppHeader, AppFooter } from "pages";
+import 'antd/dist/antd.css'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-Eventia/eventus/pullRequest?prid=9101009)
```suggestion
import "antd/dist/antd.css"
```